### PR TITLE
Update Translations.cs

### PR DESCRIPTION
**Pull request by Ambushed91**
Minor text and layout changes

### CODE REVIEW

**Comment by DurtyFree** [comment]
@Ambushed91 Exact reason for these changes?

Need some opinions from the community too

**Comment by Ambushed91** [comment]
Visual changes while output. Better read-able

File: PoGo.PokeMobBot.Logic/Common/Translations.cs [hunk 1]
```diff
@@ -187,7 +187,7 @@ public class Translation : ITranslation
                 "{0}\t- CP: {1}  IV: {2}%   [Best CP: {3}  IV: {4}%] (Candies: {5})"),
             new KeyValuePair<TranslationString, string>(TranslationString.EventItemRecycled, "{0}x {1}"),
             new KeyValuePair<TranslationString, string>(TranslationString.EventPokemonCapture,
-                "({0}) | ({1}) {2} Lvl: {3} CP: ({4}/{5}) IV: {6}% | Chance: {7}% | {8}m dist | with a {9} ({10} left). | {11}xp | {12}"),
+                "({0}) | {2}, Lvl: {3} | CP: ({4}/{5}) | IV: {6}% | Type: {1} | Chance: {7}% | Dist: {8}m | Used: {9} ({10} left) | XP: {11} | {12}"),
```
Comment (DurtyFree) [inline PoGo.PokeMobBot.Logic/Common/Translations.cs]: You shouldn't just take out a variable and ( {1} ) without considering changing the backend ( code behind that )

**Comment by DurtyFree** [comment]
Alright, read my comment @Ambushed91 :)

**Comment by Ambushed91** [comment]
@DurtyFree {1} Wasnt removed, its replaced to be a " Type"  pokemon

**Comment by DurtyFree** [comment]
Actually didn't see that.
Thx!:)

**Pull request merged**